bp/src/BPConfiguration.m

```diff
@@ -95,7 +95,7 @@ typedef NS_OPTIONS(NSUInteger, BPOptionType) {
 
     // options with no argument
     {'L', "clone-simulator", BP_MASTER | BP_SLAVE, NO, NO, no_argument, "Off", BP_VALUE | BP_BOOL , "cloneSimulator",
```

Comment:
Change default value from `"Off"` to `"On"` since it says _always on_ ;-)

Reply:
Done.